observation_portal/proposals/viewsets.py

```diff
@@ -67,6 +68,11 @@ def globallimit(self, request, pk=None):
         else:
             return Response({'errors': serializer.errors}, status=400)
 
+    @action(detail=False, methods=['get'])
+    def tags(self, request, pk=None):
```

Comment:
This endpoint is used by the frontend to get the options for the available tags that can be filtered on.